source/admin_manual/logging.rst

```diff
@@ -6,15 +6,45 @@ Dovecot Logging
 
 **Dovecot always logs a detailed error message** if something goes wrong. If it doesn't, it's considered a bug and will be fixed. However, almost always the problem is that **you're looking at the wrong log file**; error messages may be logged to a different file than informational messages.
 
+By default Dovecot logs to syslog using mail facility. Dovecot can also be configured to log to files directly.
```

Comment:
I think s/syslog/the syslog/

And possible s/logs to/uses/

Reply:
"Logs to the syslog" sounds weird to me. Maybe "logs to the syslog service"?

Reply:
"uses the syslog service with mail facility"?

Reply:
"uses the syslog service with mail facility" sounds good